### DISCUSSION TITLE
Is opentelemetry-cpp v1.15.0 compatible with Abseil as old as 2020-02-25?

### DISCUSSION THREAD

**Discussion by laharimuthyala** [open-telemetry/opentelemetry-cpp#2718] [Q&A]
I'm trying to integrate opentelemetry-cpp v1.15.0 into an application that uses an older version of Abseil. So I've set `DWITH_ABSEIL=OFF`. But I get errors (like below) on compilation - 

```
include/opentelemetry/sdk/metrics/view/meter_selector_factory.h:21:1: error: expected expression
public:

include/opentelemetry/sdk/metrics/view/meter_selector_factory.h:19:28: error: variable has incomplete type 'class OPENTELEMETRY_EXPORT'
class OPENTELEMETRY_EXPORT MeterSelectorFactory
```

Is there a way around this issue other than upgrading Abseil?

**Comment by owent** [comment]
Yes, I use otelcol-cpp v1.15 with abseil-cpp `20200225.3` and GCC 4.8 and it works well.
